app/src/main/res/layout/fragment_settings_app_advanced.xml

```diff
@@ -102,6 +105,34 @@
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/item_separator_height"
                 android:background="@color/item_separator"/>
+
+            <TextView
+                style="@style/Header"
+                android:text="@string/screenshots" />
+
+            <io.gnosis.safe.ui.settings.view.SettingItem
+                android:id="@+id/screenshot_permission"
+                android:layout_width="match_parent"
+                android:layout_height="60dp"
+                android:paddingTop="@dimen/default_margin"
+                android:paddingBottom="@dimen/default_margin"
+                android:background="@color/white"
+                app:setting_openable="false"
+                app:setting_has_switch="true"
+                app:setting_name="@string/enable_screenshots"
+                />
```

Comment:
we have closing tags on the same line as the last xml parameter elsewhere. Better to remain consistent here also